.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,34 @@
-## Description
+# Description
 
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
 Fixes # (issue)
 
-## Tested on 
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Tested on?
+
 - [ ] Windows  
-- [ ] Linux Ubuntu  
+- [ ] Linux Ubuntu 
+- [ ] Centos 
 - [ ] Mac  
 - [ ] Others  (State here -> xxx )  
 
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged
+
 Assign a reviewer to review your code.
```

Comment:
suggest removing the line of

```
Assign a reviewer to review your code
```

the PR issuer should know intuitively they have to assign a reviewer